src/main.rs

```diff
@@ -127,13 +130,28 @@ fn prune(dir: ReadDir) {
         }
 
         println!("    Pruning for lib {:?}", lib);
-        let _ = lib_paths.pop();
+        let latest = unwrap!(unwrap!(unwrap!(lib_paths.pop()).metadata()).modified());
         for lib_path in lib_paths {
-            print!("      Deleting {:?} ... ", lib_path);
-            if let Err(e) = fs::remove_file(lib_path) {
-                println!("ERROR: {:?}", e);
+            let earlier = unwrap!(unwrap!(lib_path.metadata()).modified());
+            let rm = ALLOWED_DURATION_SEC == 0
```

Comment:
why ALLOWED_DURATION_SEC == 0 since it is a constant?

Reply:
Ah kinda if someone wants to configure it to 0 to do what it was previously doing (keep 0 dups). Since this is not overly good UX one is kinda expected to change params manually and compile. So instead of bloating it with docopts or configs just did this.